lara-typescript/src/section-authoring/components/section-column.tsx

```diff
@@ -132,6 +132,16 @@ export const SectionColumn: React.FC<ISectionColumnProps> = ({
 
   const showItemPicker = () => setShowAddItem(true);
 
+  const isWrappingItem = (itemData: any) => {
+    if (
```

Comment:
You could also do this:  `return ["questionWrapper", "interactives"].indexOf(itemData.componentLabel) !== -1`

However, this does give a nice breakpoint location to debug in.